u2f-gae-demo/war/WEB-INF/appengine-web.xml

```diff
@@ -1,7 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
   <application>u2fdemo</application>
-  <version>crxjs</version>
+  
+  <!--
+    Version of the extension deployment.
+    Uncomment this if you want to deploy a server instance that uses the extension
```

Comment:
"that uses the extension that uses the U2F Chrome extension": s/that uses the extension//